index.html

```diff
@@ -1,165 +1,462 @@
+                                            name="group[33317][4]"
+                                            id="mce-group[33317]-33317-2"
+                                        /><label for="mce-group[33317]-33317-2"
+                                            >BuildGT</label
+                                        >
+                                    </li>
+                                    <li>
+                                        <input
+                                            type="checkbox"
+                                            value="8"
+                                            name="group[33317][8]"
+                                            id="mce-group[33317]-33317-3"
+                                        /><label for="mce-group[33317]-33317-3"
+                                            >Catalyst</label
+                                        >
+                                    </li>
+                                    <li>
+                                        <input
+                                            type="checkbox"
+                                            value="16"
+                                            name="group[33317][16]"
+                                            id="mce-group[33317]-33317-4"
+                                        /><label for="mce-group[33317]-33317-4"
+                                            >Horizons</label
+                                        >
+                                    </li>
+                                </ul>
+                            </div>
+                            <div id="mce-responses" class="clear">
+                                <div
+                                    class="response"
+                                    id="mce-error-response"
+                                    style="color: white"
+                                ></div>
+                                <div
+                                    class="response"
+                                    id="mce-success-response"
+                                    style="color: white"
+                                ></div>
+                            </div>
+                            <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+                            <div
+                                style="position: absolute; left: -5000px;"
+                                aria-hidden="true"
+                            >
+                                <input
+                                    type="text"
+                                    name="b_34c4c58551eca5eacf86c9479_7ac5ebfb77"
+                                    tabindex="-1"
+                                    value=""
+                                />
+                            </div>
+                            <div class="clear">
+                                <input
```

Comment:
Can you add some padding to the left of the button?
![image](https://user-images.githubusercontent.com/6752596/86190247-e3dd7100-bb11-11ea-9a13-7fe46aa5a5e1.png)